python/cudf/cudf/core/udf/masked_lowering.py

```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2021-2023, NVIDIA CORPORATION.
+# Copyright (c) 2021-2024, NVIDIA CORPORATION.
```

Comment:
not sure ow this change happened but seem to be right...

Reply:
this is the result of running pre-commit on all files, so it updated files that have not been touched by any PR for a long time so pre-commit did not run

cc: @vyasr

Reply:
It shouldn't trigger updates of files unless other parts of your git repo are out of date I believe. You may need to git fetch the remote or something. Files that you did not modify in this PR shouldn't get updated. If that's happening we can troubleshoot.

Reply:
The copyright was updated because you changed another part of the file (added lines 10-11). This update should be correct.